app/views/course/groups/new.html.slim

```diff
@@ -1,8 +1,10 @@
 - add_breadcrumb :new
 = page_header
+/ Sort course users by name to populate the dropdown list
+- course_users = (current_course.course_users - [current_course_user]).sort_by! { |course_user| course_user.name }
```

Comment:
how about `current_course.course_users.sory_by_some_order - current_course_user` ? it will be faster.